orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -26,14 +26,11 @@ import (
 	"github.com/stretchr/testify/assert"
```

Comment:
For the `swagger_protos` import line: here and elsewhere: don't alias imports unless the alias is necessary

*One asterisk to this rule: we have certain packages that we *always* import, like `merrors`. This is a quirk of our codebase, and I have a todo to rename that package to `merrors` to avoid having to do the permanent alias